tests/acceptance/Lists/ManageListsCest.php

```diff
@@ -211,6 +212,32 @@ public function cantTrashOrBulkTrashActivelyUsedList(\AcceptanceTester $i) {
     $i->waitForText('0 lists were moved to the trash.');
   }
 
+  public function cantTrashOrBulkTrashListWithForm(\AcceptanceTester $i) {
+    $listTitle = 'List with form';
+    $segmentFactory = new Segment();
+    $segment = $segmentFactory
+      ->withName($listTitle)
+      ->create();
+    $formName = 'My Form';
+    $formFactory = new Form();
+    $formFactory
+      ->withName($formName)
+      ->withSegments([$segment])
+      ->create();
+
+    $i->wantTo('Check that user can’t delete a list that is assigned to a form');
+    $i->login();
+    $i->amOnMailpoetPage('Lists');
+    $i->waitForText($listTitle, 5, '[data-automation-id="listing_item_' . $segment->getId() . '"]');
+    $i->clickItemRowActionByItemName($listTitle, 'Move to trash');
+    $i->waitForText("List cannot be deleted because it’s used for '{$formName}' form");
+    $i->seeNoJSErrors();
+    $i->checkOption('[data-automation-id="listing-row-checkbox-' . $segment->getId() . '"]');
+    $i->waitForText('Move to trash');
+    $i->click('Move to trash');
+    $i->waitForText('0 lists were moved to the trash.');
```

Comment:
I personally find it a bit confusing that we are silently ignoring the user's request to delete a few of the lists if they are still being used in a form or an e-mail. I would expect a more detailed message saying something like "X lists were moved to the trash. Unable to move one list because it is still being used in a form". My concern here is that, when deleting several lists at once, it is easy for the user to miss that some were not deleted as we are just showing a notification that indicates success. They have to know how many lists they tried to delete and compare that to the number of lists that were deleted and that we are showing in the message.

I thought it was worth mentioning this for discussion, but I'm not sure if we should do anything about it in this PR as I believe what I'm describing above applies more broadly to things that are not related to this ticket. For example, the same happens when deleting lists that are used in e-mails.

Reply:
This is a good point. I will create a new issue to address it and we can discuss there how to handle the different cases.

Reply:
Sounds good to me. Thanks!

Reply:
Issue created.